arxiv/forms/csrf.py

```diff
@@ -1,65 +1,14 @@
-"""
```

Comment:
Removing CSRF protection. 

This is only in use the the unused code from the submission-ui repo.
 
WTForms has instructions for configuring CSRF.  We can use that instead and it is easy to setup.